cmd/config-current.go

```diff
@@ -479,6 +486,27 @@ func loadConfig() error {
 	return nil
 }
 
+// getAuthProviders - returns ProviderList which contains enabled providers in serverConfig.
+// A new notification provider is added like below
```

Comment:
Did you intend "authentication" provider instead of notification here?

Reply:
copy paste issue.